<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartSettingsDialog</class>
 <widget class="QDialog" name="ChartSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Theme:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxTheme"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Legend:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxLegend"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxTitle">
       <property name="toolTip">
        <string>Shows chart title</string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAntiAliasing">
     <property name="text">
      <string>Anti-aliasing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChartSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTheme</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ChartSettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxLegend</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ChartSettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAntiAliasing</sender>
   <signal>clicked(bool)</signal>
   <receiver>ChartSettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTitle</sender>
   <signal>clicked()</signal>
   <receiver>ChartSettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changed()</slot>
 </slots>
</ui>
